src/sdk/odp/client/raw_storage_client.py

```diff
@@ -99,8 +102,14 @@ def list_paginated(
 
         if cursor:
             params["page"] = cursor
-        if limit:
-            params["limit"] = limit
+        if page_size:
+            params["page_size"] = page_size
+        if limit & limit != 1000:
```

Comment:
Maybe convert the limit to a default value of None and checking if it is None or not.

Reply:
ya. that can be done.